acceptancetests/assess_caas_bootstrap.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+"""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+import os
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+)
+
+from jujucharm import (
+    local_charm_path
+)
+
+
+__metaclass__ = type
+
+
+log = logging.getLogger("assess_caas_bootstrap")
+
+
+def assess_caas_bootstrap(client):
+    # Deploy k8s bundle to spin up k8s cluster
+    repository_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'repository')
+    bundle = local_charm_path(charm='bundles-kubernetes-core-lxd.yaml',
+                              repository=repository_dir, juju_ver=client.version)
+    client.deploy_bundle(bundle, static_bundle=True)
+    # Wait for the deployment to finish.
+    client.wait_for_started()
```

Comment:
You'll probably need to wait for workloads too, perhaps there might be something more you'll need to wait for? Perhaps workloads is enough.

wait_for_started and wait_for_workloads are the default 'wait for' methods used, but BaseCondition allows one to write specific "wait for this status within a timeout". See jujupy.wait_condition.UnitInstallCondition as an example of a specific status check.